Makefile.am

```diff
@@ -85,3 +86,9 @@ desktopdir = $(datadir)/applications
 desktop_in_files = geany.desktop
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
+
+appdata_in_files = geany.appdata.xml.in
+
+appdatadir = $(datadir)/appdata
+nodist_appdata_DATA = geany.appdata.xml
```

Comment:
this should probably use `$(appdata_in_files:.xml.in=.xml)` just like the desktop one above.